tool/utils.ts

```diff
@@ -2,9 +2,9 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+const fetch = require('make-fetch-happen').defaults();
```

Comment:
Please use an `import` statement here and then assign fetch below the imports.

e.g. something like:
```ts
import {defaults as initMakeFetchHappen} 'make-fetch-happen';
// other imports
const fetch = initMakeFetchHappen();
```